src/main/resources/application.properties

```diff
@@ -12,17 +12,15 @@ server.servlet.contextPath=/Geoweaver
 logging.level.root=FATAL
 logging.level.org.springframework=FATAL
 logging.level.com.gw=FATAL
-logging.level.org.hibernate=FATAL
+logging.level.org.hibernate=DEBUG
 
+# import the external configuration file if exists
+spring.config.import=optional:file:${HOME}/geoweaver/application.properties
```

Comment:
Is this recorded somewhere in the documentation? how do people know where to change?